src/common/updating/updating.cpp

```diff
@@ -0,0 +1,241 @@
+#include "pch.h"
+
+#include "version.h"
+
+#include "updating.h"
+
+#include <msi.h>
+#include <common/common.h>
+#include <common/json.h>
+#include <common/version.h>
+#include <common/settings_helpers.h>
+#include <common/winstore.h>
+#include <common/notifications.h>
+
+#include <winrt/Windows.Web.Http.h>
+#include <winrt/Windows.Web.Http.Headers.h>
+#include <winrt/Windows.Storage.Streams.h>
+#include <winrt/Windows.Management.Deployment.h>
+#include <winrt/Windows.Networking.Connectivity.h>
+
+#include "VersionHelper.h"
+
+namespace
+{
+    const wchar_t POWER_TOYS_UPGRADE_CODE[] = L"{42B84BF7-5FBF-473B-9C8B-049DC16F7708}";
+    const wchar_t DONT_SHOW_AGAIN_RECORD_REGISTRY_PATH[] = L"delete_previous_powertoys_confirm";
+    const wchar_t USER_AGENT[] = L"Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.2; WOW64; Trident/6.0)";
+    const wchar_t LATEST_RELEASE_ENDPOINT[] = L"https://api.github.com/repos/microsoft/PowerToys/releases/latest";
+    const wchar_t MSIX_PACKAGE_NAME[] = L"Microsoft.PowerToys";
+    const wchar_t MSIX_PACKAGE_PUBLISHER[] = L"CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US";
+}
+
+namespace localized_strings
```

Comment:
shouldn't this be in an RC file for loc?

Reply:
yes, though since it's a static library, it has the same issue as `common`, see https://github.com/microsoft/PowerToys/issues/1311